Webex Connect Flows/v3.0/Sample/Usage of Screen Pop in Flows/README.md

```diff
@@ -0,0 +1,20 @@
+
+
+# Screen Pop Sample Flows Overview :
+- Screen Pop feature allows you to trigger screen pop based on various parameters of the ongoing conversation.
+- In Screen pop node we need to define url and query parameters which will be passed as query string to a third party application which is opened as screen pop.
+- This example demonstrates the usage of Screen Pop node using Task Close Flow but any workflow can use the Screen Pop Node.
+- Please refer to the documentation for more details: https://help.imiconnect.io/docs/screen-pop
+
+The folder includes the following sample flow for Screen Pop :
+## Task Close Flow With Screen Pop 
+- This flow will send a screen pop message to the agent contact on WebexCC once the contact is closed.
+- Screen Pop will consist of the following things
```

Comment:
"Following things" sounds little informal. "Screen Pop Node consists of the following configuration fields"???